src/common/constantModule.js

```diff
@@ -0,0 +1,21 @@
+const constant = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const defaultState = [
+    {
+      id: Math.floor(Math.random() * 1000),
+      title: 'Somoye and his projects',
+      category: 'Action',
+    },
+    {
+      id: Math.floor(Math.random() * 1000),
```

Comment:
- [ ] Consider wrapping the randomId generator as a function and calling it in each case.